.github/workflows/lighthouse-ci.yml

```diff
@@ -0,0 +1,71 @@
+name: Google Lighthouse
+
+on:
```

Comment:
I'm not sure this is correct, I believe the `on` here should be of `pull_request`, since this should be updated on every Pull Request.